index.js

```diff
@@ -1,17 +1,35 @@
 
-var { benchBranch, benchmarkRuntime } = require("./bench");
+const { benchBranch, benchmarkRuntime } = require("./bench");
+
+let allowedUsers = process.env.ALLOWED_USERS || "";
+allowedUsers = allowedUsers.split(",").map((item) => Number(item.trim())).filter((item) => !Number.isNaN(item) && item !== 0);
```

Comment:
simplified (Number already trims and both NaN and 0 are false values, so identity check is enough)

```suggestion
allowedUsers = allowedUsers.split(",").map(Number).filter(item => item);
```

Reply:
Nice. I know that there was a simpler way.